app/code/Magento/Ui/view/base/web/js/lib/core/class.js

```diff
@@ -61,7 +61,8 @@ define([
     Class = createConstructor({
 
         /**
-         * Entry point to the initialization of consturctors' instance.
+         * Entry point to the initialization of 
```

Comment:
Looks like auto-replace failed here, please fix, do commit amend and force push to the same branch.

Reply:
Done